docs/Contributing/game-admin.rst

```diff
@@ -0,0 +1,170 @@
+local working area. Skip the ``clone`` steps for Freeciv21.
+
+Now clone the games repository:
+
+.. code-block:: sh
+
+  ~/$ cd GitHub
+  ~/GitHub$ git clone git@github.com:longturn/games.git
+
+
+You will be prompted for your SSH key to complete the ``clone``. You should have a local :file:`games` directory when
+complete.
+
+Games Repo Structure
+====================
+
+The Games repository has a logical structure::
+
+  - [Game Number] Directory (e.g. LT86)
+    - [Game Number].serv
+    - [Game Number].sh
+    - db.conf
+    - env.sh
+    - players.serv
+
+    - data/[Game Number] Directory (e.g. data/LT86)
+      - buildings.ruleset
+      - cities.ruleset
+      - effects.ruleset
+      - game.ruleset
+      - government.ruleset
+      - nations.ruleset
+      - parser.lua
+      - script.lua
+      - styles.ruleset
+      - techs.ruleset
+      - terrain.ruleset
+      - units.ruleset
+
+
+Here are some notes/edits needed for each file.
+
+#. ``[Game Number].serv`` -- This is the primary server configuration file. You can edit any and all parameters to setup
+   the game the way you want. Refer to the :doc:`/Manuals/Server/index`, and for help with the map generator, refer to
+   :doc:`/Manuals/Advanced/map-generator`.
+#. ``[Game Number].sh`` -- Edit the top three values for the ``Game Number`` and set the TCP Port for the game.
+#. ``db.conf`` -- Edit for the ``Game Number``
+#. ``env.sh`` -- Edit the ``FREECIV_DATA_PATH`` value for the ``Game Number``
+#. ``players.serv`` -- This is the list of players for the game. You can get the list from the game page on the main
+   longturn.net website. Refer to :doc:`/Manuals/Advanced/players` for more information. You can also refer to other
+   recent games for help with the format.
+#. The files in ``data/[Game Number]`` are standard ruleset files. You can copy the ruleset files from our common
+   `ruleset repository <https://github.com/longturn/LTT-LTX>`_, or create your own custom ruleset. We have Sim and
+   Aviation as current third-part rulesets we play games with. For more information on ruleset files, refer to
+   :doc:`/Modding/index`.
+
+
+Setup a New Game
+================
+
+Now that you know the basics of how the game files on the repository work, let us move to the steps to admin a new game.
+
+You must first have been granted game admin rights on the main longturn website. The website uses Django as the back-end,
+so managing game pages on the site is very simple. If you do not have access, please ask on the main Longturn Discord
+server.
+
+#. Login at https://longturn.net
+#. On the lower left, click "Admin Site".
+#. On the site administration panel on the left, click ``+ Add`` on the row for *Games*.
+#. Give it a name, such as LT86. The name must be unique and can not have been used before.
+#. Fill in the game description field. See below on some more notes related to the game description.
+#. Select the game mode: Team Game, Teamless, Experimental.
```

Comment:
Maybe explain the meaning of Experimental (=rules are expected to change during the game)